zingolib/src/lightclient.rs

```diff
@@ -463,24 +465,43 @@ impl LightClient {
             .iter()
         {
             LightClient::tx_summary_matcher(&mut summaries, *txid, transaction_md);
-            let tx_fee = transaction_md.get_transaction_fee();
-            let (block_height, datetime, price) = (
-                transaction_md.block_height,
-                transaction_md.datetime,
-                transaction_md.price,
-            );
-            if transaction_md.is_outgoing_transaction() {
-                summaries.push(ValueTransfer {
-                    block_height,
-                    datetime,
-                    kind: ValueTransferKind::Fee { amount: tx_fee },
-                    memos: vec![],
-                    price,
-                    txid: *txid,
-                });
-            }
+
+            let tx_fee_result = transaction_md.get_transaction_fee();
+            match tx_fee_result {
+                Ok(tx_fee) => {
+                    if transaction_md.is_outgoing_transaction() {
```

Comment:
Do we care about the fee associated with a `!transaction_md.is_outgoing_transaction()` ?

Reply:
do we? our consumers currently show fees, but maybe its unnecessary

Reply:
however they are free to not show the fees if they wish